x-pack/plugins/siem/public/components/page/hosts/hosts_table/translations.ts

```diff
@@ -18,6 +18,10 @@ export const FIRST_SEEN = i18n.translate('xpack.siem.hostsTable.firstSeen', {
   defaultMessage: 'First Seen',
 });
 
+export const LAST_SEEN = i18n.translate('xpack.siem.hostsTable.lastSeen', {
```

Comment:
Missing [i18n message types](https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#defining-type-for-message).